### PR TITLE
feat(chronos): add date/time handling module

### PR DESCRIPTION
**Pull request by ziadsawalha**
This ports existing code and tests (and adds new
tests to achieve 100% coverage) from existing
projects.

### CODE REVIEW

File: simpl/chronos.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+`chronos.get_time_string()`. Date/times are always UTC. We leave it to clients
+to convert to local time.
+
+Our APIs consitently return date/time in the API_FORMAT format.
+"""
+
+import datetime
+import time
+
+API_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
```
Comment (samstav) [inline simpl/chronos.py]: If this format, and these utilities, comply with [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601), maybe leave a note/comment/documentation about that.

Comment (ziadsawalha) [inline simpl/chronos.py]: Updated the commit message :-)

Comment (ziadsawalha) [inline simpl/chronos.py]: See 1450e59ad4477f8133d5a3765af9c47682a663ca

**Comment by samstav** [comment]
LGTM

**Pull request merged**